src/main/js/portal/main/components/ui/Message.tsx

```diff
@@ -2,17 +2,24 @@ import React, { Component, ReactNode } from 'react';
 
 type Props = {
 	title: string
-	onclick: () => void
+	findData: () => void
+	restorePriorCart?: () => void
 }
 
 export default class Message extends Component<Props>{
 	render() {
+		const buttonRestorePriorCart = this.props.restorePriorCart ? 
+			(<button className="btn btn-lg btn-warning d-block mx-auto my-3" onClick={this.props.restorePriorCart}>
```

Comment:
I would probably use a `btn-outline-secondary` here to differentiate from regular next actions that use blue or yellow buttons.